main.md

```diff
@@ -54,29 +54,32 @@ documents that support selective disclosure of claim values.
 The JSON-based content of JSON Web Signatures (JWS) as defined in [@!RFC7515] is
 secured against modification using digital signatures. A consumer of a JWS
 document that has checked the document's signature can safely assume that the
-contents of the document have not been modified.  However, anyone receiving a
-JWS document can read all contents of the document. 
-
-For example, a common use case is that the signed document represents a user's
-identity credential, created by an issuer. The issuer includes the user's public
-key or a reference thereto. To proof their identity to a verifier, the user can
-then send the issuer-signed credential plus a signature over some
-transaction-specific values, the so-called proof. It is signed using the user's
-private key. This demonstrates possession of the private key, and by extension,
-the identity of the user. 
-
-The problem is, that using this approach, the user has to release the full
-issuer-signed credential to the verifier. The credential is often created once
-and can then be used for many transactions. Thus, it is in the user's interest
-that the credential creates many user attributes which can be disclosed
-selectively to verifiers.
-
-This document describes a format for JWS documents that support selective
-disclosure (SD-JWS) including a format for proofs.
-
-It is important to note that while user identity credentials of natural persons
-are common use cases, the mechanisms defined in this document can be used for
-any other use case as well. 
+contents of the document have not been modified.  This means that anyone 
+receiving a JWS document can read all contents of the document because the all 
+of the JWT claims are secured using one signature.
+
+This is not a problem in a federated identity use case when the user has given 
+the Client authorization to access certain resources, and the Authorization 
+Server returns to the Client only the claims the user has consented to release.
+
+However, in a use case involving verifiable credentials where the user is 
+presenting a JWS signed by the issuer at multiple Clients over a period of time
```

Comment:
I opened an issue on this, but my 2 cents are...
from my understanding, JAdES is close enough to JWS, it should be included. 
I do not see the benefit of including LD-Proofs and group signatures, since the whole point of this work was to enable selective disclosure without using those signature formats...

Reply:
I agree with @Sakurann.

Reply:
You're right. See comment: https://github.com/oauthstuff/draft-selective-disclosure-jwt/issues/12

We can continue using only JWS.